clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/GoogleInfrastructureProvider.groovy

```diff
@@ -71,18 +74,19 @@ class GoogleInfrastructureProvider extends AgentSchedulerAware implements Search
       return result + [
           healthCheckLink: backendService.attributes.healthCheckLink as String,
           sessionAffinity: backendService.attributes.sessionAffinity as String,
-          affinityCookieTtlSec: backendService.attributes.affinityCookieTtlSec as String
+          affinityCookieTtlSec: backendService.attributes?.affinityCookieTtlSec as String,
```

Comment:
Would have already NPE'd on the previous line; can't see reason to use `?.` here and below.